collector.proto

```diff
@@ -12,89 +12,90 @@ option optimize_for = CODE_SIZE;
 import "google/protobuf/timestamp.proto";
 
 message SpanContext {
-    uint64 trace_id = 1;
-    uint64 span_id = 2;
-    map<string, string> baggage = 3;
+  uint64 trace_id = 1;
+  uint64 span_id = 2;
+  map<string, string> baggage = 3;
 }
 
 message KeyValue {
-    string key = 1;
-    oneof value {
-	string string_value = 2;
-	int64 int_value = 3;
-	double double_value = 4;
-	bool bool_value = 5;
-    }
+  string key = 1;
+  oneof value {
+    string string_value = 2;
+    int64 int_value = 3;
+    double double_value = 4;
+    bool bool_value = 5;
+    string json_value = 6;
```

Comment:
let's add a comment explaining that the `json_value` MUST be well-formed JSON (i.e., no hard string truncations allowed).